### DISCUSSION TITLE
set renderFieldContent sending default render

### DISCUSSION THREAD

**Discussion by Naveena1321** [clientIO/joint#2777] [Q&A] [answered]
### Introduction

I have renderFieldContent inside my inspector.create function where i have a switch case to return input for custom option type but how to have default case for it

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
The default field is used when the function returns `undefined`.

You might have to return an empty array `[]` in TypeScript.

**Reply by Naveena1321** [reply]
Cannot read properties of undefined (reading 'closest');
 and i know it is fixed in the new version but we are on 4.0.0 
 is there any path for this

**Reply by kumilingus** [reply, marked as answer]
There was a bug. Upgrade to `v4.0.1` please.
